internal/compiler/langtype.rs

```diff
@@ -764,20 +771,46 @@ impl EnumerationValue {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub struct LengthConversionPowers {
+    pub rem_to_px_power: Option<i8>,
```

Comment:
why an Option.  if there is none it should sipmly be 0. 
And the return value is already in an Option anyway.

Reply:
Still valid. This doesn't need to be option it seems